setup.py

```diff
@@ -38,6 +38,9 @@
     if os.path.exists(pjoin(d, '__init__.py')):
         packages.append(d[len(here)+1:].replace(os.path.sep, '.'))
 
+package_data = {'nb_conda_kernels': [pjoin('logos', 'python', '*'),
```

Comment:
Again, haven't run this, but: if we are installing the *actual* kernel packages, they should include their own logos, and we needn't ship them, or?